.github/workflows/build.yml

```diff
@@ -2,8 +2,7 @@ name: Build
 
 on:
   push:
-    ignore-branches:
-      - main
+    branches-ignore: [main]
```

Comment:
thank god to the GitHub Actions extension to catch this!

https://spacelift-io.slack.com/archives/C0475GQ1E5P/p1681895483909389